src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedStyles = {};
+
+  let propertyArray = sourceString.replace(/[\n]/gi, '').split(';');
+
```

Comment:
Wow! It looks great! But this topic is about array iteration methods, not regex 😮. And even Misha said it's strongly not recomended to use it. I can approve it but I'm sure that mentors will tell you to remake it using array iteration methods.

Reply:
It`s just made code a lot simpler and in that case I thought that it would be great option to make string look in a proper way with that kind of solution ) Also I have used methods where it was necessary

Reply:
It's okay to use `regex` here)